packages/langium/src/references/linker.ts

```diff
@@ -133,13 +173,20 @@ export class DefaultLinker implements Linker {
         return description ?? this.createLinkingError(refInfo);
     }
 
+    getCandidates(refInfo: ReferenceInfo): AstNodeDescription[] | LinkingError {
+        const scope = this.scopeProvider.getScope(refInfo);
+        const descriptions = scope.getElements(refInfo.reference.$refText).distinct(desc => `${desc.documentUri}#${desc.path}`).toArray();
+        return descriptions.length > 0 ? descriptions : this.createLinkingError(refInfo);
```

Comment:
The multiplicity symbol `*` in the current grammar proposal could imply that zero link targets is okay and not a linking error. Shall we make a difference between `*` and `+` here?